### ISSUE TITLE
unhandled top-level exception: $exn

### ISSUE THREAD

**Issue by simseokhyun** [MinaProtocol/mina#4326]
2020-02-03 18:33:25 UTC [Fatal] Unhandled top-level exception: $exn
Generating crash report
        exn: "(monitor.ml.Error\n  (Failure\n    \"Failed to connect to Kademlia process: RPC #2 failed: \\\"initializing helper error: Map log file. Path=/home/dannyjr95/.coda-config/coda_net2/libp2p_helper/libp2p-peerstore-v0/000000.vlog. Error=exec format error\\\"\")\n  (\"Raised at file \\\"stdlib.ml\\\", line 33, characters 17-33\"\n    \"Called from file \\\"src/deferred1.ml\\\", line 17, characters 40-45\"\n    \"Called from file \\\"src/job_queue.ml\\\" (inlined), line 131, characters 2-5\"\n    \"Called from file \\\"src/job_queue.ml\\\", line 171, characters 6-47\"\n    \"Caught by monitor coda\"))"

  ☠  Coda daemon crashed. The Coda Protocol developers would like to know why!

  Please:
    Open an issue:
      <https://github.com/CodaProtocol/coda/issues/new>

    Briefly describe what you were doing and attach the crash report /home/dannyjr95/.coda-config/coda_crash_report_2020-02-03_18-33-25.630943.tar.gz

**Comment by emberian** [comment]
Thank you for the report, and sorry for the long processing delay! I think this is the same as #4254 @simseokhyun, can you confirm that you were running this inside of WSL?

**Comment by simseokhyun** [comment]
Hi ember,

Yes, I was running this in WSL. I tried to run docker in WSL, officially
made an inquiry to microsoft and got the answer that the feature will be
available in WSL2 which will be released within this year.
I decided to run coda through Google Cloud and all of the problems now
don't apply to me!
Thank you for recognizing my problem.
Me and my fellow peers now use GCP, so I think we won't face the problem :)

Thank you!

Sincerely,
Danny.

2020년 5월 29일 (금) 오전 3:38, ember arlynx <notifications@github.com>님이 작성:
